andela/bp-esa-frontend
# #168094491 Fix automation status

**What does this PR do?**
This PR adds changes to fix the status of automations. When any of the Slack, Email, or Noko automations fail, the status should clearly reflect a failed automation. The status should indicate `Success` only if all automations have passed.

**Description of the task to be completed?**
- fix the automation status in the card view
- fix the automation status in the details modal

**How should this be manually tested?**
- Clone and set up the project using the instruction in the README.
- $ git checkout to `bg-fix-automation-status-168094491` branch which contains the changes.
- Use $ yarn start to start the application
- Filter different automations for `Slack`, `Email`, `Noko` success and failure.
The status should only be successful if all automations pass.

**What are the relevant pivotal tracker stories?**
[#168094491](https://www.pivotaltracker.com/story/show/168094491)

**Screenshots**
N/A
